### PR TITLE
-XX:+CompactStrings is enabled by default on jdk11+

### LINKED ISSUE #1160: `-XX:+CompactStrings` is enabled by default on jdk11+
**Issue or pull request number:**
https://github.com/eclipse-openj9/openj9/pull/16452

**Overview:**
`-XX:+CompactStrings` is enabled by default on jdk11+.

**Release target:**
0.41

**Applies to the following JDK versions:**
jdk11 and later

**Applies to the following platforms:**
All platforms

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1160

Updated that the `-XX:+CompactStrings` option is enabled by default on Java 11 and later in the related topics.

Closes #1160
Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the documents. Please check. Thanks!

File: docs/xxcompactstrings.md [hunk 1]
```diff
@@ -40,4 +40,6 @@ Further details are available at [JEP 254: Compact Strings](https://openjdk.org/
 | `-XX:+CompactStrings`    | Enable `String` compression  |         |
 | `-XX:-CompactStrings`    | Disable `String` compression | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
```
Comment (pshipton) [inline docs/xxcompactstrings.md:41]: It's still marked as disabled by default in the table, this should be updated.

Comment (Sreekala-Gopakumar) [reply docs/xxcompactstrings.md:41]: But, it is enabled by default only for Version 11 and later, right? So for version 8 and other versions before 11, It is still disabled, by default, isn't it?

Comment (pshipton) [reply docs/xxcompactstrings.md:41]: Can we reflect that in the table with an image for `jdk11+`? I think we've done that elsewhere. 
There are no other versions than 8 before 11.

Comment (Sreekala-Gopakumar) [reply docs/xxcompactstrings.md:41]: So, should I give tick marks in both the rows but with different icons - 1st row with jdk11+ icon and second row with jdk8 icon?

Comment (pshipton) [reply docs/xxcompactstrings.md:41]: Yes

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - Please check and confirm. Thanks!

**Comment by pshipton** [comment]
Looks good. The start of xxcompactstrings shows "Java 8+" and I'm not sure why since this doc only supports 8+. Can you pls remove it.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
> Looks good. The start of xxcompactstrings shows "Java 8+" and I'm not sure why since this doc only supports 8+. Can you pls remove it.

Done. Please check. Thanks!

**Comment by pshipton** [comment]
Jenkins doc stage

**Review by pshipton — APPROVED** [review]

**Pull request merged**